shift.sh

```diff
@@ -0,0 +1,31 @@
+jscodeshift \
```

Comment:
Nop, just wip "snapshot". Will force all kinds of stuff later, use the force you know 🌠